### PR TITLE
Adds concepts and procedures for LTI

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2298](https://openedx.atlassian.net/browse/DOC-2298), [DOC-2248](https://openedx.atlassian.net/browse/DOC-2248), [DOC-2295](https://openedx.atlassian.net/browse/DOC-2295)

Adds LTI authentication concepts, and procedures to set up and test the non-anonymous workflow.
### Date Needed

This feature is already released. Please review as soon as feasible.
### Reviewers

Any one of these SMEs required, >1 preferred:
- [x] Subject matter expert: @fredsmith 
- [ ] Subject matter expert: @cpennington 
- [x] Subject matter expert: @ormsbee 
- [x] Doc team review (sanity check): @srpearce 
- [ ] Product review: @ebporter

FYI: @bradenmacdonald @sarina @jibsheet @mhoeber 
### Sandbox

See  [OPS-997](https://openedx.atlassian.net/browse/OPS-997). For those with credentials, the procedures can be reviewed/tested in Edge.
### Testing
- [X] make html ran without unexpected errors
### Post-review
- [ ] Squash commits
- [ ] Add description to release notes task as a comment
- [ ] Update change log

### CODE REVIEW

File: en_us/install_operations/source/configuration/lti/tpa_lti.rst [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+   available. For example, use the Django administration console to open the
+   **Change Provider Configuration (LTI)** page.
+
+#. Use a separate browser window or tab to open the `IMS LTI 1.1 Consumer
+   Launch`_ page.
+
+#. As the **Launch URL**, enter your base URL followed by ``/auth/login/lti/``.
+   For example, ``http://{your_URL}/auth/login/lti/``.
+
+#. Copy the **Lti consumer key** value, and then on the IMS LTI 1.1 Consumer
```
Comment (srpearce) [inline en_us/install_operations/source/configuration/lti/tpa_lti.rst:92]: Nit: "Consumer Launch" should be bold.

File: en_us/install_operations/source/configuration/lti/authentication_options_lti.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+After you :ref:`configure your edX instance as an LTI tool provider<Configuring
+an edX Instance as an LTI Tool Provider>`, no further configuration is needed
+on your Open edX system for this user authentication flow.
+
+******************************
+Open edX User Authentication
+******************************
+
+The first time a learner encounters an Open edX resource in a course, he is is
```
Comment (srpearce) [inline en_us/install_operations/source/configuration/lti/authentication_options_lti.rst]: Extra "is" at end of line

**Comment by srpearce** [comment]
Doc review complete.

**Comment by mhoeber** [comment]
this helped me understand this much more, thanks. :+1:

**Comment by fredsmith** [comment]
:+1:

**Comment by ormsbee** [comment]
:+1:

**Pull request merged**